src/hearings/containers/hearing-actuals/hearing-actual-add-edit-summary/hearing-actual-add-edit-summary.component.html

```diff
@@ -1,4 +1,147 @@
-<exui-hearing-actuals-summary
-  [template]="template"
-  [mode]="mode">
-</exui-hearing-actuals-summary>
+<a class="govuk-back-link" href="javascript:history.back()" (click)="onBack()">Back</a>
+<main class="govuk-main-wrapper govuk-main-wrapper--auto-spacing" id="main-content" role="main">
+  <div class="govuk-grid-row" *ngIf="hearingActualsMainModel">
+    <div class="govuk-grid-column-two-thirds">
+      <span class="govuk-caption-l"></span>
+      <h1 class="govuk-heading-l">Hearing details</h1>
+      <dl class="govuk-summary-list">
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">Hearing date</dt>
+          <dd class="govuk-summary-list__value">{{hearingActualsMainModel.hearingActuals.actualHearingDays[0].hearingDate | date:"dd MMMM yyyy"}}</dd>
```

Comment:
could this hearingActualsMainModel.hearingActuals.actualHearingDays[0] put into the variable in ts as it's used many times in the template

Reply:
I have now updated the code.